2-Webpack/scripts/localstorage_manager.js

```diff
@@ -0,0 +1,44 @@
+export class LocalStorageManager {
+
+  constructor() {
+    if (LocalStorageManager._instance) {
+      return LocalStorageManager._instance;
+    }
+    LocalStorageManager._instance = this;
```

Comment:
why singleton?

Reply:
I don't think it makes sense to allow more than 1 instance of this manager, as it only connects to the local storage. Its instances will never have different states. It's just a bunch of APIs bundled together to a class, maybe it shouldn't even be a class :D

Reply:
Done in the last commit - [2599afd](https://github.com/seanoch/karate-kid-kickstart/pull/2/commits/2599afd2b5a12ceb56e7f0766918827470632518)